### ISSUE TITLE
Support Jobs delete artifacts API

### ISSUE THREAD

**Issue by jawscout** [gitlab4j/gitlab4j-api#493]
GitLab added this call in `11.9` https://docs.gitlab.com/ee/api/jobs.html#delete-artifacts

**Comment by gmessner** [comment]
@jawscout 
This functionality has been released in 4.14.10, the following methods have been added to ``JobApi```:
```
public Job playJob(Object projectIdOrPath, Integer jobId)
public Job keepArtifacts(Object projectIdOrPath, Integer jobId)
```

Please verify and close this issue when you have time.

**Comment by gmessner** [comment]
@jawscout 
Since this functionality has been released I am closing this issue.  If you find any problems please feel free to reopen it.
